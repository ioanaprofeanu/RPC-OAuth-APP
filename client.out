PERMISSION_DENIED
v53teNBmWTXjUd0 -> mNBIv3N_5DSeGEt
PERMISSION_GRANTED
OPERATION_NOT_PERMITTED
TOKEN_EXPIRED
dvBj3mU5WXTtN0e -> 5U_GNSDjBv3IEdm
OPERATION_NOT_PERMITTED
PERMISSION_GRANTED
REQUEST_DENIED
pPHXta6YpMas3RJ -> S_ENG6pHtDaIXYP
RESOURCE_NOT_FOUND
