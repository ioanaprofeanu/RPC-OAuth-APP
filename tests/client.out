e49HDdxTqL7dZya -> aZy4TLHxqded9D7
d10i68x86f4dfN2 -> 2Nx0id84f6d168f,x8f02fd4iN661d8
PERMISSION_GRANTED
PERMISSION_GRANTED
TOKEN_EXPIRED
PERMISSION_GRANTED
OPERATION_NOT_PERMITTED
TOKEN_EXPIRED
REQUEST_DENIED
TOKEN_EXPIRED
TOKEN_EXPIRED
fi4N018fx668dd2 -> x1628id60N8ffd4,48d2606dx1fNi8f
PERMISSION_GRANTED
OPERATION_NOT_PERMITTED
TOKEN_EXPIRED
TOKEN_EXPIRED
